code/modules/admin/verbs/podlauncher.dm

```diff
@@ -6,6 +6,9 @@
 		return
 
 	var/datum/podlauncher/P = new(usr)
+	if(!P.bay || !P.podarea)
+		return
```

Comment:
```suggestion
		to_chat(user, "No bay or podarea found.")
		return
```

Reply:
If you look below that's already handled and user is messaged what part is erroring out.